.github/workflows/ci.yml

```diff
@@ -42,6 +42,10 @@ jobs:
         node-version: ${{ matrix.node }}
     - name: Install Packages
       run: npm install
+      if: ${{ !startswith(matrix.node, '16') }}
+    - name: Install Packages
+      run: npm install --force
```

Comment:
I think its ok to use `--force`, also we can add a comment here `remove it after v8.0.0 stable release`

Reply:
Done in 972238c